lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
@marcparadise Just want to clarify that I needed to add **{ }** to that API call (see below)
rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users", **{** "username" => user **}** )

@jkeiser Here is the extra information when run in debug mode

```
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::JSONInput#handle_request
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::JSONOutput#handle_request
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::CookieManager#handle_request
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Decompressor#handle_request
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Authenticator#handle_request
[2015-02-26T03:19:03+00:00] DEBUG: Signing the request as pivotal
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::RemoteRequestID#handle_request
[2015-02-26T03:19:03+00:00] DEBUG: Initiating POST to https://poliva-chef12-standalone-thor.poliva.dev.altus.bblabs/organizations/blackberry/users
[2015-02-26T03:19:03+00:00] DEBUG: ---- HTTP Request Header Data: ----
[2015-02-26T03:19:03+00:00] DEBUG: Content-Type: application/json
[2015-02-26T03:19:03+00:00] DEBUG: Accept: application/json
[2015-02-26T03:19:03+00:00] DEBUG: Accept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-SIGN: algorithm=sha1;version=1.0;
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-USERID: pivotal
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-TIMESTAMP: 2015-02-26T03:19:03Z
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-CONTENT-HASH: 8wwvnFFgxZMkq71ibtabEtJuUOM=
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-AUTHORIZATION-1: cmEQ1n3QojJGOJeVOCM1tdQCADSpzVhsjZQVS8tP7olNJbrAVxGMPKrw5ld9
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-AUTHORIZATION-2: BVPcXAMRWGQzf8ynPxMon/aK8avZ2t5dMosv0HGplGW5LKPBGMeFjNOJ3h5U
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-AUTHORIZATION-3: ZNxPozGWzEgxJn4hZfXHUXiigJukQ4z6Z2RO3Lsy/l0+wQgPkYodXZrmAoKK
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-AUTHORIZATION-4: dQ53HBqxWGVTIop6GeMEBZ+0tPe71M6tV+XFZU+ESMaITkidB64wfuA/IrTK
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-AUTHORIZATION-5: AJNQEUac1QCzGdwxRPBqMtRHrBbejdnVvFYPNv1YSpKB2QteYmfA6Ie+N/62
[2015-02-26T03:19:03+00:00] DEBUG: X-OPS-AUTHORIZATION-6: CchhpoAm//Qaw9mcGxTCV/5lfTLCrvGvm9U265qaeA==
[2015-02-26T03:19:03+00:00] DEBUG: HOST: poliva-chef12-standalone-thor.poliva.dev.altus.bblabs:443
[2015-02-26T03:19:03+00:00] DEBUG: X-REMOTE-REQUEST-ID: d3bb0f93-b335-4c7d-930f-473e23c71a7e
[2015-02-26T03:19:03+00:00] DEBUG: Content-Length: 21
[2015-02-26T03:19:03+00:00] DEBUG: ---- End HTTP Request Header Data ----
[2015-02-26T03:19:03+00:00] DEBUG: ---- HTTP Status and Header Data: ----
[2015-02-26T03:19:03+00:00] DEBUG: HTTP 1.1 201 Created
[2015-02-26T03:19:03+00:00] DEBUG: server: ngx_openresty/1.4.3.6
[2015-02-26T03:19:03+00:00] DEBUG: date: Thu, 26 Feb 2015 03:19:03 GMT
[2015-02-26T03:19:03+00:00] DEBUG: content-type: application/json
[2015-02-26T03:19:03+00:00] DEBUG: content-length: 0
[2015-02-26T03:19:03+00:00] DEBUG: connection: close
[2015-02-26T03:19:03+00:00] DEBUG: x-ops-api-info: flavor=cs;version=12.0.0;oc_erchef=1.2.0
[2015-02-26T03:19:03+00:00] DEBUG: location: http://poliva-chef12-standalone-thor.poliva.dev.altus.bblabs/organizations/blackberry/users/poliva
[2015-02-26T03:19:03+00:00] DEBUG: ---- End HTTP Status/Header Data ----
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::RemoteRequestID#handle_response
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Authenticator#handle_response
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::Decompressor#handle_response
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::CookieManager#handle_response
[2015-02-26T03:19:03+00:00] DEBUG: Chef::HTTP calling Chef::HTTP::JSONOutput#handle_response
[2015-02-26T03:19:03+00:00] INFO: HTTP Request Returned 201 Created:

    ================================================================================
    Error executing action `create` on resource 'chef_organization[blackberry]'
    ================================================================================

    Chef::Exceptions::JSON::ParseError
    ----------------------------------
    Top level JSON object must be a Hash or Array. (actual: NilClass)
```